src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -1344,7 +1357,7 @@ void VM_Version::get_processor_features() {
 #endif // COMPILER2 && ASSERT
 
 #ifdef _LP64
-  if (supports_avx512ifma() && supports_avx512vlbw() && MaxVectorSize >= 64) {
+  if ((supports_avx512ifma() && supports_avx512vlbw() && MaxVectorSize >= 64) || (supports_avxifma() && MaxVectorSize >= 32))  {
```

Comment:
This could be:
if ((supports_avx512ifma() && supports_avx512vlbw()) || (supports_avxifma()))  {

Reply:
Please see the code updated as suggested.